plugin/ranger.vim

```diff
@@ -17,19 +17,24 @@ function! s:RangerMagic(path) " {{{
 
   " Create a new temporary file name
   let g:ranger_tempfile = tempname()
-  let opts = ' --choosefiles=' . shellescape(g:ranger_tempfile) . ' ' . shellescape(a:path)
+  if g:ranger_cd_mode
+    let l:choose_arg = 'choosedir'
+  else
+    let l:choose_arg = 'choosefiles'
+  endif
+  let l:opts =  printf('--%s=%s %s', l:choose_arg, g:ranger_tempfile, shellescape(a:path))
```

Comment:
Why remove the shellescape on `g:ranger_tempfile`?

Reply:
Was done accidentally, now I fixed it.